dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingSoftQueue.java

```diff
@@ -45,7 +45,7 @@ public SimpleRotatingSoftQueue(T[] elements) {
      * @return true
      */
     @Override
-    public boolean add(T element) {
+    public synchronized boolean add(T element) {
```

Comment:
not clear why this is synchronized (when other methods accessing elements are not)